src/utils/general.js

```diff
@@ -0,0 +1,17 @@
+const runInformation = 'This command needs to be invoked inside a plugin directory.';
+
+/**
+ * Creates the description for the command
+ * 
+ * @param {string} description  the main description
+ * @param {string} inDirectory  whether this command should be invoked inside a plugin directory or not
+ * @returns {string} the updated command
+ */
+module.exports.createDescription = (description, inDirectory) => {
+  description = `${description.trim('.')}.`;
```

Comment:
`.trim('.')`?

Reply:
trim doesn't take in args 🤔and it trims whitespace so i'm not sure what you're doing here?

Reply:
oops :p I thought I did 😂